snippets/form-password-session-pages.liquid

```diff
@@ -0,0 +1,64 @@
+{% comment %}
+
+  This snippet is using for rendering the form on Edit password, Accept invitation and Reset password pages.
+
+  form_name - "string" optional, the name of the form
+  password_label - "string" required, the label of the first field
+  password_confirmation_label - "string" required, the label of the second field
+  button_label - "string" required
+
+  Usage:
+
+  {%- render 'form-password-session-pages',
+      form: form,
+      form_name: "edit_password",
+      password_label: section.settings.password_label,
+      password_confirmation_label: section.settings.password_confirmation_label,
+      button_label: section.settings.change_my_password
+  -%}
+
+{% endcomment %}
+
+{%- if password_label != blank -%}
+  <div class="{% if form.errors.password %}account-fieldset--error{% endif %}">
+    <label for="user_password" class="account-fieldset__label">{{ password_label }}</label>
+    <input
+      value="{{- form.password -}}"
+      placeholder=" "
```

Comment:
Not really sure why we need it here with such value

Reply:
It looks weird, but it's required for the label animation:
<img width="396" alt="image" src="https://github.com/user-attachments/assets/3873bee7-0672-4766-9d45-156b514686fc">
<img width="382" alt="image" src="https://github.com/user-attachments/assets/01a5473d-51c3-4b22-86ed-284e1ab41582">

Reply:
Ah, got ya